src/backend/joanie/core/utils/emails.py

```diff
@@ -0,0 +1,44 @@
+"""Utility to prepare email context data variables for installment payments"""
+
+from django.conf import settings
+
+from stockholm import Money
+
+from joanie.core.enums import PAYMENT_STATE_PAID, PAYMENT_STATE_REFUSED
+
+
+def prepare_context_data(order, installment_amount, product_title, refused: bool):
```

Comment:
```suggestion
def prepare_context_data(order, installment_amount, product_title, payment_refused: bool):
```

IMO `refused` is not explicit enough.